mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,355 @@
+		ARG 0 input
+		ARG 1 minInclusive
+		ARG 3 maxInclusive
+		ARG 5 whenInRange
+		ARG 6 whenOutOfRange
+	METHOD m_nqecyxgz cacheOnce (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_ojhmwxkk shift2 (Lnet/minecraft/unmapped/C_rxrtfjrt;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 offsetNoise
+	METHOD m_ovaexbma add (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_pfuzzaai constant (D)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 value
+	METHOD m_qwpxedjm map (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_mdkmcezi$C_ucbebgzr$C_vgrinicv;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+		ARG 1 type
+	METHOD m_rrapcnud slide (Lnet/minecraft/unmapped/C_ulvgadxi;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 settings
+		ARG 1 input
+	METHOD m_skodlkvx cacheAllInCell (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_sswjlstj mapFromUnitTo (Lnet/minecraft/unmapped/C_kzwapyud;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 value
+	METHOD m_tofdnidr endIslands (J)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 random
+	METHOD m_tykuysov interpolated (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function
+	METHOD m_unqecqgl shift1 (Lnet/minecraft/unmapped/C_rxrtfjrt;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 offsetNoise
+	METHOD m_vgtncips max (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 function1
+		ARG 1 function2
+	METHOD m_zjrjjrgd shift0 (Lnet/minecraft/unmapped/C_rxrtfjrt;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 offsetNoise
+	CLASS C_aovxcpjj ClampedGradientY
+		FIELD f_brhvizcz fromY I
+		FIELD f_fxihhpnv fromValue D
+		FIELD f_nncsoykg toValue D
+		FIELD f_yydgzbsh toY I
+		METHOD <init> (IIDD)V
+			ARG 1 fromY
+			ARG 2 toY
+			ARG 3 fromValue
+			ARG 5 toValue
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_czahebwa fromValue ()D
+		METHOD m_khgjowhl fromY ()I
+		METHOD m_mitcetii toY ()I
+		METHOD m_nearmskl toValue ()D
+	CLASS C_ihcxcncj MultiplyOrAdd
+		FIELD f_axuyyfjb argument D
+		FIELD f_dhjeyvpa type Lnet/minecraft/unmapped/C_mdkmcezi$C_ihcxcncj$C_oasfbkwq;
+		FIELD f_lotenpvv input Lnet/minecraft/unmapped/C_kzwapyud;
+		FIELD f_ltmwezav maxValue D
+		FIELD f_nfrfpmxe minValue D
+		METHOD <init> (Lnet/minecraft/unmapped/C_mdkmcezi$C_ihcxcncj$C_oasfbkwq;Lnet/minecraft/unmapped/C_kzwapyud;DDD)V
+			ARG 1 type
+			ARG 2 input
+			ARG 3 minValue
+			ARG 5 maxValue
+			ARG 7 argument
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD m_acrcrutr argument ()D
+		METHOD m_nyhirayq type ()Lnet/minecraft/unmapped/C_mdkmcezi$C_ihcxcncj$C_oasfbkwq;
```

Comment:
If this class is a record, it's missing the methods for `input`, `minValue` and `maxValue`. If not, these methods should be named `getArgument` and `getType`

Reply:
It seems like there are multiple records missing methods, or the methods they do have should have a `get` prefix